perf/perf.py

```diff
@@ -461,7 +466,15 @@ def grpc_create_database(router_id, writer_id):
             },
             'routing_config': {'sink': {'kafka': {}}},
             'worker_cleanup_avg_sleep': '500s',
-            'writing': '127.0.0.1:9093',
+            'write_buffer_connection': {
+                'direction': 1,
```

Comment:
IIRC you could also use a string here:

```suggestion
                'direction': 'DIRECTION_WRITE',
```